packages/exo/src/exo-tools.js

```diff
@@ -402,14 +398,43 @@ export const defendPrototype = (
   }
 
   for (const prop of methodNames) {
+    const originalMethod = behaviorMethods[prop];
+    const { shiftedMethod } = {
+      shiftedMethod(...args) {
+        return originalMethod(this, ...args);
+      },
+    };
+    const behaviorMethod = thisfulMethods ? originalMethod : shiftedMethod;
+    // TODO some tool does not yet understand the `?.[` syntax
```

Comment:
Specifically `node -r esm` which `agoric deploy` scripts might still use, but it’s not clear to me that any old deploy scripts depend on `@endo/exo`.

Reply:
God to know. Added note pointing here. Resolving.